iso/cOS-local.yaml

```diff
@@ -1,6 +1,8 @@
 packages:
   rootfs:
   - system/cos
+  - system/initrd
```

Comment:
Just a reminder, maybe that was committed accidentally: we need to make `system/cos` depend on those two instead before merging, so we have one image only for the upgrade.

Reply:
Yes I know, it was just to speed up the testing. This is clearly still work in progress, but I wanted to start showing it just in case there was stuff to discuss. And, at least, I wanted to make a PR that eventually can be build locally and produces something close to the intended result of this PR.